x-pack/plugins/cases/public/components/create/form_context.test.tsx

```diff
@@ -149,8 +149,7 @@ const waitForFormToRender = async (renderer: Screen) => {
   });
 };
 
-// Failing: See https://github.com/elastic/kibana/issues/146394
-describe.skip('Create case', () => {
+describe('Create case', () => {
```

Comment:
This test was skipped for a long time and was not updated after latest custom fields feature update.